katcp/resource.py

```diff
@@ -894,9 +892,7 @@ def handle_update(sensor, reading):
             # This handler is called whenever a sensor update is received
             try:
                 assert sensor is self
-                cond_matched = condition_test(reading)
-                status_matched = reading.status == status or status is None
```

Comment:
Cool, thanks for the clarification!

On 4 August 2016 at 09:47, Theuns Alberts <notifications@github.com> wrote:

> In katcp/resource.py
> <https://github.com/ska-sa/katcp-python/pull/159#discussion_r73476314>:
>
> > @@ -894,9 +892,7 @@ def handle_update(sensor, reading):
> >              # This handler is called whenever a sensor update is received
> >              try:
> >                  assert sensor is self
> > -                cond_matched = condition_test(reading)
> > -                status_matched = reading.status == status or status is None
>
> Agree, I wanted to make sure we did not want to add status checking in the
> non-callable case as well. Given that we agree that the satus-checking is
> not necessary its superflous to discuss this anymore.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ska-sa/katcp-python/pull/159/files/856d61c24464b667b00629859206ed0c4f022aa7#r73476314>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAX70Am0Gk9tE2Mx_Ecq019SluSVUnJsks5qcZkugaJpZM4JYodi>
> .
>